README.md

```diff
@@ -75,9 +82,26 @@ The [update_gene_symbols.py](https://github.com/d3b-center/D3b-DGD-Collaboration
 python3 /Users/brownm28/Documents/git_repos/D3b-DGD-Collaboration/scripts/update_gene_symbols.py -g hgnc_complete_set_2021-06-01.txt -f PVS1.level -z GENE level -u GENE -o results --explode_records 2> old_new.log
 ```
 With `results` used to replace `PVS1.level` file. Recommend references for this workflow can be obtained [here](https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-references/files/#q?path=d3b_diskin_pathogenicity).
+
 ### InterVar Classification Workflow
 This workflow is a critical component in generating scoring metrics needed to classify pathogenicity of variants.
 Documentation for this can be found [here](docs/INTERVAR_WF.md)
 ### AutoPVS1
 An additional pathogenicity scoring tool, run on the VEP-annotated input.
-Documentation for this can be found [here](https://github.com/d3b-center/D3b-autoPVS1/tree/v2.0.0#readme)
\ No newline at end of file
+Documentation for this can be found [here](https://github.com/d3b-center/D3b-autoPVS1/tree/v2.0.0#readme)
+
+### Optional Inputs
+As mentioned above, the preprocessing workflow can add an additional annotation
+ - `annotation_vcf`: hg38 chromosome-formatted vcf file with multi-allelics split. If provided bcftools will add annotation from the specified columns for each variant that matches
```

Comment:
- `annotation_vcf`: hg38 chromosome-formatted vcf file with multi-allelics split. If provided, BCFtools will add annotation from the specified columns for each variant that matches